src/Core/Grid/Definition/Factory/LogGridDefinitionFactory.php

```diff
@@ -229,10 +231,11 @@ public function getFilters()
             ->add(
                 (new Filter('actions', SearchAndResetType::class))
                     ->setTypeOptions([
-                        'attr' => [
-                            'data-url' => $this->resetActionUrl,
-                            'data-redirect' => $this->redirectionUrl,
+                        'reset_route' => 'admin_common_reset_search_by_filter_id',
```

Comment:
I think you could clean the constructor (since the arguments are not used any more) and the service definition no?

It might be considered a BC break although it's arguable since this is a final class not destined to be extended nor constructed manually, so at worst it's a small one

Reply:
Fixed